tests/test_package.py

```diff
@@ -31,7 +31,7 @@ def test_sign_file(monkeypatch):
     package = package_file.PackageFile(
         filename=filename,
         comment=None,
-        metadata=pretend.stub(name="deprecated-pypirc"),
+        metadata=dict(name="deprecated-pypirc", version="1.2.3"),
```

Comment:
The `metadata` argument to the `PackageFile` constructor is not anymore the object produced by `pkginfo` (I'm too lazy to check how it is called) but a `packaging.metadata.RawMetadata` typed dict. `version` is a required metadata entry, thus the code in the `PackageFile` constructor assumes it is there. These changes simply reflect that.

Reply:
@dnicolodi in that case, it might be better to explicitly use that dict class instead of just `dict(...)`

```suggestion
        metadata=packaging.metadata.RawMetadata(name="deprecated-pypirc", version="1.2.3"),
```

Reply:
Suggested changes applied.